cli/commands.go

```diff
@@ -38,7 +38,7 @@ import (
 
 const (
 	commentSeparator  = "##### Comments go below this line #####"
-	defaultRPCTimeout = time.Second * 10
+	defaultRPCTimeout = time.Second * 20
```

Comment:
Why changing the RPC timeout?

Reply:
The time out should be 20 as normal latency is AVG 20